### PR TITLE
[windows] use PostMessage to replace SendMessage

### PR DESCRIPTION
**Pull request by PatriceJiang**
- speedup font loading 
- make CCLOG non-block

### CODE REVIEW

File: cocos/base/CCConsole.cpp [hunk 1]
```diff
@@ -108,10 +108,12 @@ namespace {
         if (Director::getInstance()->getOpenGLView())
         {
             HWND hwnd = Director::getInstance()->getOpenGLView()->getWin32Window();
-            SendMessage(hwnd,
-                        WM_COPYDATA,
-                        (WPARAM)(HWND)hwnd,
-                        (LPARAM)(LPVOID)&myCDS);
+            // use non-block version of SendMessage
```
Comment (crazyhappygame) [inline cocos/base/CCConsole.cpp:111]: Could you double check docs:
https://docs.microsoft.com/en-us/windows/desktop/api/winuser/nf-winuser-sendnotifymessagea
Are you sure about SendNotifyMessageA?
"If the window was created by the calling thread, SendNotifyMessage calls the window procedure for the window and does not return until the window procedure has processed the message"
Maybe PostMessage should be used?

Comment (PatriceJiang) [reply cocos/base/CCConsole.cpp:111]: `PostMessage` seems more "non-block" than `SendNotifyMessage`, thanks for your correcting .

**Comment by minggo** [comment]
Could you please describe why it can speedup and not block the engine?

**Comment by PatriceJiang** [comment]
Since `SendMessage` is blocking, when loading font,  `BitmapDC::setFont` sometimes takes `500ms+` to wait the invoke result. 
`PostMessage` does not wait, that makes it run faster.

**Comment by minggo** [comment]
Please send a PR to v3 too.

**Comment by PatriceJiang** [comment]
Okay

**Pull request merged**